java/core/src/java/org/apache/orc/DataReader.java

```diff
@@ -54,9 +54,18 @@ BufferChunkList readFileData(BufferChunkList range,
   /**
    * Releases buffers created by readFileData. See readFileData javadoc.
    * @param toRelease The buffer to release.
+   *
+   * @deprecated Use {@link #releaseAllBuffers()} instead. This method was
+   * incorrectly added and shouldn't be used anymore.
```

Comment:
Oh, I realized that this message is copied here (https://github.com/apache/orc/pull/1574/files#r1290619552). Actually, I want to give you an example format instead of the message content.

The following message is not proper to this method, `releaseBuffer`, because it was a required API. This PR had better claim that the issue exists in the implementation instead of the API's existence. Could you revise a little more?
> This method was incorrectly added